### PR TITLE
revoke completed

### PR DESCRIPTION
**Pull request by RavindraDesai042**

### CODE REVIEW

File: apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl [hunk 1]
```diff
@@ -148,3 +148,41 @@
...
+                <form class="form-inline">
+                    <label class="form-group mx-sm-3 mb-2 ">
+                        <input type="radio" name="revokeNow" value="revokeNow" onclick=$("#revokeValidToDiv").hide() >
+                        Revoke Now
+                    </label>
+                    <label class="form-group mx-sm-3 mb-2 ">
+                        <input type="radio" name="revokeNow" value="revokeLater"  onclick=$("#revokeValidToDiv").show() checked>
+                        Revoke Later
+                    </label>
+                    <input type="hidden" name="subscriptionId" id="subscriptionId" value="${subscriptionId}"/>
```
Comment (RavindraDesai042) [inline apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl]: value="${subscriptionId}" if i remove this, sometimes subscriptionId is coming empty. If I add this it works fine but in the pop up error comes

File: apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl [hunk 2]
```diff
@@ -148,3 +148,41 @@
...
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <form class="form-inline">
+                    <label class="form-group mx-sm-3 mb-2 ">
+                        <input type="radio" name="revokeNow" value="revokeNow" onclick=$("#revokeValidToDiv").hide() >
+                        Revoke Now
+                    </label>
+                    <label class="form-group mx-sm-3 mb-2 ">
```
Comment (RavindraDesai042) [inline apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl]: from the radio button in ajax, always same value (revokeNow) comes

File: apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl (hunk 1 above)
Comment (harishbnaritha) [reply apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl]: $subscriptionId is never going to work... this Id is coming from JS, $(subscriptionId) is a server side ftl syntax

File: apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl (hunk 2 above)
Comment (harishbnaritha) [reply apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl]: Ok I will check

**Pull request merged**